runtime/runc_jailer.go

```diff
@@ -393,6 +408,25 @@ func (j *runcJailer) copyFileToJail(src, dst string, mode os.FileMode) error {
 	return nil
 }
 
+// copyDirectoryToJail will copy all of the files and the parent directory to
+// the given destination folder, dst. This will also set the proper ownership
+// to the files and folders.
+func (j *runcJailer) copyDirectoryToJail(src, dst string) error {
+	output, err := exec.Command("cp", "-rf", src, dst).CombinedOutput()
+	if err != nil {
+		return errors.Wrapf(err, "failed to copy directory %q to %q: %s", src, dst, output)
+	}
+
+	filepath.Walk(dst, func(path string, info os.FileInfo, err error) error {
```

Comment:
I think we need to check the error from filepath.Walk.

Reply:
Will fix